arcgis-ios-sdk-samples/Augmented reality/Display scenes in tabletop AR/README.md

```diff
@@ -16,10 +16,10 @@ You'll see a feed from the camera when you open the sample. Tap on any flat, hor
 
 1. Create an `ArcGISARView` and add it to the view.
     * Note: this sample uses content in the WGS 84 geographic tiling scheme, rather than the web mercator tiling scheme. Once a scene has been displayed, the scene view cannot display another scene with a non-matching tiling scheme. To avoid that, the sample starts by showing a blank scene with an invisible base surface. Touch events will not be raised for the scene view unless a scene is displayed.
-2. Listen for ARKit tracking state updates with `arView.arSCNViewDelegate` and provide feedback to the user as necessary.
-3. When tracking is ready, wait for the user to tap, then use `arView.setInitialTransformation(using:)` to set the initial transformation, which allows you to place the scene. This method uses ARKit's built-in plane detection.
+2. Listen for ARKit tracking state updates with `ArcGISARView.arSCNViewDelegate` and provide feedback to the user as necessary.
+3. When tracking is ready, wait for the user to tap, then use `ArcGISARView.setInitialTransformation(using:)` to set the initial transformation, which allows you to place the scene. This method uses ARKit's built-in plane detection.
 4. Create and display the scene. To allow you to look at the content from below, set the base surface navigation constraint to `none`.
-5. Set the clipping distance property of the AR view. This will clip the scene to the area you want to show.
+5. To create a realistic tabletop mapping experience, set the scene's base surface opacity to 0. This will ensure that only the target scene content is visible.
```

Comment:
It doesn't appear that this iOS sample sets this clipping distance yet (see issue 1678 in our other repo). Might we need to remove step 5 altogether, if this is the case?

Reply:
Oops. I think you can disregard my comment altogether. I see that the new change to step 5 sets the surface's opacity, and doesn't concern the clipping distance. In this case, all LGTM. I'll change my response to an approval.

Reply:
Thanks! For this line I changed from the design doc a bit, to better describe the iOS implementation.